docs/central-install-digital-ocean.rst

```diff
@@ -565,6 +565,104 @@ DKIM is a security trust protocol which is used to help verify mail server ident
+
+   .. code-block:: console
+
+     $ cd central
+
+   .. code-block:: console
+
+     $ nano .env
+
+   .. code-block:: bash
+
+     OIDC_ENABLED=true
+     OIDC_ISSUER_URL=my-issuer-url
+     OIDC_CLIENT_ID=my-client-id
+     OIDC_CLIENT_SECRET=my-client-secret
+
+#. Build and restart all containers.
+
+   .. code-block:: console
+
+     $ docker compose build && docker compose stop && docker compose up -d
+
+Two Accounts: Central and the Identity Provider
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you enable SSO, users will use their account on the identity provider to log into Central. However, users will still have a Central account that is separate from the identity provider. A Central account is not automatically created for each account on the identity provider. Instead, a Central administrator will need to create a Central account for each user of the identity provider who should be allowed to log into Central.
+
+Central users will be able to change their display name shown in Central and to choose a different name from what is shown in the identity provider. However, because Central identifies users by their email address, most users will not be allowed to change their email address. Only a Central Administrator will be able to change the email address associated with a Central account. That will be needed if a user's email address changes within the identity provider. In that case, an Administrator will need to manually change the user's email address within Central to match their new address in the identity provider.
+
+If a Central Administrator changes their own email address to one that does not match the identity provider, they may lose access to Central. If they are the only Administrator, they will need to disable SSO, set a password :ref:`by command line <central-command-line-user-set-password>`, log in, change their email address, then re-enable SSO.
+
+Logout is not centralized, which means that when a user logs out of Central, that will not log them out of the identity provider. Conversely, when a user logs out of the identity provider, that will not log them out of Central. If a user logs out of Central, then goes to log back in, they may find that login is nearly instantaneous if they are still logged into the identity provider. That is, they may find that they are not required to log into the identity provider again in order to log into Central.
+
+Enabling SSO in an Existing Installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to enable SSO for an existing Central installation, even if the installation has existing users. Follow the instructions above to enable SSO. Because Central identifies users by their email address, the email address associated with a Central account must match the address in the identity provider. If the email address does not match, the user will not be able to log in.
```

Comment:
I think this is all true. We've been able to toggle SSO on and off on dev servers. There was related discussion at https://github.com/getodk/central-backend/pull/910#discussion_r1269624813 that wasn't fully resolved. However, I understand the flow better now than when I left those comments.